presto-main/src/main/java/com/facebook/presto/tdigest/TDigestUtils.java

```diff
@@ -56,6 +60,20 @@ private static double weightedAverageSorted(double x1, double w1, double x2, dou
         return Math.max(x1, Math.min(x, x2));
     }
 
+    public static boolean isTDigestFunction(String name, List<TypeSignatureProvider> parameters)
+    {
+        if (name.equals(NAME)) {
+            return true;
+        }
+
+        for (TypeSignatureProvider parameter : parameters) {
```

Comment:
The right way of checking parameter type is to use the `Type` apis. This is not a reliable way.